randovania/data/json_data/prime1.txt

```diff
@@ -720,7 +725,7 @@ Asset id: 3538349
 Quarantine Monitor
```

Comment:
Quarantine Monitor connections should require morph ball

Reply:
Does setting it to a Morph Ball Door not accomplish that?

Reply:
I was told it's still important to have for those connections.

Reply:
If the area in front of the door is a morph ball tunnel, then yeah it's nice to have this requirement.
Though it only really matters if that door is the spawn point of the area, since it implies you need Morph Ball to spawn there.